RELEASE.md

```diff
@@ -78,9 +76,6 @@ File the following CLOUDP tickets for deploying the new release (you may need to
 - "Release Database Tools X.Y.Z to CM/OM" with a component of "Cloud Manager Upgrade" and assigned team of "Automation"
 - "Release Database Tools X.Y.Z to Atlas" with a component of "Atlas Upgrade" and assigned team of "Atlas Triage"
 
-#### Close Release Ticket
-Move the JIRA ticket tracking this release to the "Closed" state.
```

Comment:
hm, looks like this was deleted but not moved later on in the doc as I think you intended!

Reply:
Oops, yes, I missed that!